projects/showcase/cypress/e2e/set-style.cy.ts

```diff
@@ -40,6 +40,7 @@ describe('Set style', () => {
         });
 
         it('Then I should see the original map image', () => {
+          cy.wait(3_000);
```

Comment:
Use the driver API, the idea is to be as less cypress spcific in the tests as possible and use the driver pattern.